Adjust logic for video change

luigi311 opened the pull request:
Prevents out of bounce when scrolling to the right

Stops and resets the video when you change to the left or right

luigi311 committed:
media_view: Prevent out of bounce

Prevent out of bounce when scrolling to the right

Signed-off-by: Luis Garcia <git@luigi311.com>

luigi311 commented:
looks like this keeps playing everything in the background. Need to adjust

Joas329 commented on furios_gallery/media_view.py:
You probably want to call stop_video() in on_page_changed() as well since that handles swipe gestures and update_media_ is for the gallery buttons only rn

luigi311 replied:
moved the stop call to on_page_changed and used previous index to get the page. Seems to be working correctly on my side.

Joas329 replied:
Works perfectly

luigi311 committed:
Stop video on change

Stops and resets the video when you change to the next video
to prevent them from running in the background

Signed-off-by: Luis Garcia <git@luigi311.com>
